shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Utxow.hs

```diff
@@ -310,9 +316,12 @@ utxoWitnessed =
         TRC (UtxoEnv slot pp stakepools genDelegs, u, tx)
 
 instance
-  ( CryptoClass.Crypto c,
-    DSignable c (Hash c EraIndependentTxBody)
+  ( Era era,
+    ShelleyBased era,
+    STS (UTXOW era),
```

Comment:
I think this constraint is causing CI to time out in https://hydra.iohk.io/build/4513046/nixlog/1.
The test case in question:
```haskell
testCase "don't spend from a bootstrap address" testBootstrapNotSpending
```